src/base/bittorrent/torrenthandle.h

```diff
@@ -432,6 +432,8 @@ namespace BitTorrent
         QString m_category;
         QSet<QString> m_tags;
         bool m_hasSeedStatus;
+        int m_uploadLimit;
+        int m_downloadLimit;
```

Comment:
Seems those are unnecessary.

Reply:
But it will save fastresume even if the value didn't change unless you mean there is another way?

Reply:
>you mean there is another way?

Yes, use the getter functions to check for current value.

Reply:
Doh! thanks :P